api_tests/tests/siren_schema.ts

```diff
@@ -1,19 +1,25 @@
+/**
+ * @ledger bitcoin
+ * @ledger ethereum
+ */
+
 import { oneActorTest, twoActorTest } from "../src/actor_test";
+import SwapFactory from "../src/actors/swap_factory";
+import { sleep } from "../src/utils";
 import "../src/schema_matcher";
-import * as sirenJsonSchema from "../siren.schema.json";
 import * as sirenRootJsonSchema from "../root.schema.json";
+import * as sirenSwapJsonSchema from "../swap.schema.json";
 import { siren } from "comit-sdk";
 import axios from "axios";
-import { createDefaultSwapRequest } from "../src/utils";
 import { Actor } from "../src/actors/actor";
 import * as swapPropertiesJsonSchema from "../swap.schema.json";
-import { Rfc003Actor } from "../src/actors/rfc003_actor";
+import { SwapResponse } from "../src/payload";
 
 // ******************************************** //
 // Siren Schema tests                                 //
 // ******************************************** //
 
-async function assertValidSirenDocument(
+async function assertValidSirenSwapDocument(
```

Comment:
If you look closely on what this function does, I think you will come to the conclusion that it is pretty useless because we already have the link to the swap outside now, thanks to the new API :)

Reply:
I think we can just inline the single assertion that the document is valid siren.